### PR TITLE
bug(371): Make homepage prettier

### PR DESCRIPTION
**Pull request by tkyi**
- Add some css back in
- Fixup homepage layout

Related to https://github.com/screwdriver-cd/screwdriver/issues/371

Current:
screen shot 2016-12-02 at 1 25 44 pm
New:
screen shot 2016-12-02 at 2 24 25 pm

### CODE REVIEW

**Review by stjohnjohnson — APPROVED** [review]

File: docs/css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    border-width: 0px;
+}
+
+.homepage .row p {
+    width: 100%;
+    font-weight: 300;
+    font-size: 16px;
+}
+
+/*@media (min-width: 768px) {
```
Comment (stjohnjohnson) [inline docs/css/homepage.css:65]: commented out?

File: docs/css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    height: 62px;
+    width: 50px;
+    margin: 0;
+    padding: 0;
+}
+
+.homepage .top h1 {
+    color: #003abc;
+    font-size: 26px;
+    font-weight: 300px;
```
Comment (petey) [inline docs/css/homepage.css]: font weight doesn't have units. h1 should have font-weight: 600

File: docs/index.md [hunk 3]
```diff
@@ -1,14 +1,29 @@
+<div class="top">
+    <img src="/assets/sd_icon.png">
+    <h1>Welcome to Screwdriver!</h1>
+    <p>We've split documentation into 3 distinct sections:</p>
+</div>
 
-# Welcome to Screwdriver!
+<div class="row">
+    <div class="col-xs-6 col-md-4 about">
+        <h2>About</h2>
```
Comment (petey) [inline docs/index.md]: should the header topics match the order of the topics in the homepage? or the other way around?

File: docs/css/homepage.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+.homepage .top img {
+    height: 62px;
+    width: 50px;
+    margin: 0;
+    padding: 0;
+}
+
+.homepage .top h1 {
+    color: #003abc;
```
Comment (petey) [inline docs/css/homepage.css]: I keep thinking that these headings look like links.
Can you try this with `color: #26282a;`?

**Review by petey — CHANGES_REQUESTED** [review]

File: docs/css/homepage.css (hunk 1 above)
Comment (tkyi) [reply docs/css/homepage.css:65]: It was making everything look funky in my laptop, but I'm not sure if the min-width just needs to be changed?
screen shot 2016-12-02 at 2 09 31 pm

File: docs/index.md (hunk 3 above)
Comment (tkyi) [reply docs/index.md]: fixed

**Review by petey — APPROVED** [review]

**Comment by petey** [comment]
👍 

cat_sing_a_song

**Pull request merged**